src/main/java/com/synopsys/integration/alert/channel/jira/actions/JiraDistributionTestAction.java

```diff
@@ -111,10 +153,33 @@ public DistributionEvent createChannelTestEvent(final String jobId, final FieldA
         return new DistributionEvent(jobId, channelName, RestConstants.formatDate(new Date()), providerName, formatType, MessageContentGroup.singleton(messageContent), fieldAccessor);
     }
 
-    private boolean areTransitionsConfigured(FieldAccessor fieldAccessor) {
-        Optional<String> optionalResolveTransitionName = fieldAccessor.getString(JiraDescriptor.KEY_RESOLVE_WORKFLOW_TRANSITION).filter(StringUtils::isNotBlank);
-        Optional<String> optionalReopenTransitionName = fieldAccessor.getString(JiraDescriptor.KEY_OPEN_WORKFLOW_TRANSITION).filter(StringUtils::isNotBlank);
-        return optionalResolveTransitionName.isPresent() && optionalReopenTransitionName.isPresent();
+    private JiraMessageResult createAndSendMessage(String jobId, FieldAccessor fieldAccessor, ItemOperation operation, String messageId) throws IntegrationException {
+        logger.debug("Sending {} test message...", operation.name());
+        final DistributionEvent resolveIssueEvent = createChannelTestEvent(jobId, fieldAccessor, operation, messageId);
+        JiraMessageResult messageResult = getDistributionChannel().sendMessage(resolveIssueEvent);
+        logger.debug("{} test message sent!", operation.name());
+        return messageResult;
+    }
+
+    private Map<String, String> validateTransition(JiraTransitionHelper jiraTransitionHelper, String issueKey, String transitionName, String transitionKey, String statusCategoryKey) throws IntegrationException {
```

Comment:
Using the return type of map feels strange here. Someone is passing in the map key and all you're doing is possibly creating a map with a single entry of that key to a String. I feel that this method should return a String (perhaps optional) and let the user map the data if need be.

Reply:
I was under the impression that our validate operations modify or return a map.

Reply:
I'll convert it to Optional though, since that will have the smallest impact.